src/test/scala/uk/gov/hmrc/test/ui/cucumber/stepdefs/CommonStepDef.scala

```diff
@@ -73,6 +73,57 @@ class CommonStepDef extends BaseStepDef {
     driver.findElement(By.linkText(expectedUrl._1)).getAttribute("href") should include (expectedUrl._2)
   }
 
+  Then( """^the user should see the correct (.*) text$""") { ( text: String) =>
```

Comment:
again not sure if this is needed as we dont really need to check values on the screen